pyproject.toml

```diff
@@ -29,6 +29,7 @@ PyYAML = "^5"
 requests = "^2"
 structlog = ">=19"
 tenacity = "^5"
+pytest-isort = "^1.3.0"
```

Comment:
Had to upgrade this bc pytest was failing without this.

Reply:
This should not be added as a dependency. It should go in the dev-dependencies section.